tests/test_peak_merging.py

```diff
@@ -51,12 +51,18 @@ def test_replace_merged(intervals, merge_instructions):
 
 
 @hypothesis.given(fake_hits,
-                  hypothesis.strategies.integers(min_value=0, max_value=100))
+                  hypothesis.strategies.integers(min_value=0, max_value=int(1e18)),
+                  hypothesis.strategies.integers(min_value=0, max_value=100),
+                  hypothesis.strategies.integers(min_value=1, max_value=2),
+                  )
 @hypothesis.settings(deadline=None)
-def test_add_lone_hits(hits, peak_length):
+def test_add_lone_hits(hits, time_offset, peak_length, dt):
     peak = np.zeros(1, dtype=strax.peak_dtype())
+    peak['time'] = time_offset
+    hits['time'] += time_offset
     peak['length'] = peak_length
-    peak['dt'] = 1
+    hits['area'] = 1
```

Comment:
@WenzDaniel if we had realized this - the test would have worked. Now it always added zeros.